### DISCUSSION TITLE
Trace log setting

### DISCUSSION THREAD

**Discussion by ripamontigiorgio** [rappen/PluginTraceViewer#174] [Ideas]
Hi,
in my opinion, disabling Trace Log on organization by a yes/no popup is quite dangerous. Disabling traces should only be left to an administrator modifying System Setting on the organization.

Regards,
Giorgio

**Comment by rappen** [comment]
We know it's dangerous. Either to keep it on (might create a lot of data) and to keep it off (to not getting traces anymore).
From suggest from Microsoft we have added these prompts suggests you to think hard about what the status should be left.

It is possible to set the setting on/off in this tool, like in many tools we have to make it easier to trace.
Note that this is impossible if the user does not have the correct roles. This tool does not crack the security.
